src/pages/Header/Header.scss

```diff
@@ -107,11 +109,99 @@
   }
 }
 
-
 //buttons
 
-.singIn-button {
-  border: 0px;
-  color: $link-color;
-  padding-right: 0px;
-}
\ No newline at end of file
+.header-cart {
+  display: flex;
+}
+
+.btn {
+  font: $main_font;
+  font-weight: 600;
+  margin-left: 40px;
+  padding: 8px 20px 8px 20px;
+  border-radius: 5px;
+  display: flex;
+  align-content: center;
+  flex-wrap: wrap;
+}
+
+.btn-sifarish {
+  border: 1px $orange-theme solid;
+  background-color: $orange-theme;
+  color: white !important;
+  height: auto;
+  line-height: 24px;
+}
+
+.btn-beyan {
+  border: 1px #28a745 solid;
+  background-color: #28a745;
```

Comment:
you must use variables for your colors everywhere)

Reply:
Okay but is there a point of using variables if that color is used once?

Reply:
you might have this color in some other components) and sometimes websites change its color scheme)